README.md

```diff
@@ -1,15 +1,37 @@
 # cosc-website
 
-/index
-/about
-/coc
-/privacy
-/djangocamp
+![alt text](./img/cosc.png)
```

Comment:
Use a better alt-text say the below one, and use 
```suggestion
![COSC banner](img/banner.svg)
```

Reply:
Ok sure will update the following changes